cmd/kyma/provision/aks/opts.go

```diff
@@ -1,4 +1,4 @@
-package gcp
+package aks
```

Comment:
please double-check, moved from gcp to aks?

Reply:
Github got confused as the files are identical, but it does not create a problem.